radboudumc-theme/sass/_typography.scss

```diff
@@ -33,11 +55,64 @@ h1.display-title {
   margin-bottom: 1em;
 }
 
-
 // First header on each page
 .page-title {
   font-size: 3rem;
+  font-weight: 800;
+
+  margin-bottom: 1.5rem;
+  font-size: 2rem;
+}
+
+p a { 
+  font-weight: 800;
+}
+
+h3 a {
   font-weight: 300;
+}
 
-  margin-bottom: 1em;
+div.container > p,
+div.page-content > p {
+  margin-bottom: 0.75em;
+}
+
+@media (min-width: 768px) {
+  div.container > p,
+  div.page-content > p,
+  div.page-content > ol,
+  div.page-content > ul {
+    padding-right: 5em;
+  }
+}
+@media (min-width: 992px) {
+  div.container > p,
+  div.page-content > p,
+  div.page-content > ol,
+  div.page-content > ul {
+    padding-right: 10em;
+  }
+}
+@media (min-width: 1200px) {
+  div.container > p,
+  div.page-content > p,
+  div.page-content > ol,
+  div.page-content > ul {
+    padding-right: 20em;
+  }
+}
+
+nav a {
+  font-weight: 300;
+}
+
+// A bit more whitespace for lists
+ul, ol {
+  margin-top: 0.25rem;
+  padding-left: 1.5em;
+}
+
+ul li,
+ol li {
+  margin-bottom: 0.25rem;
```

Comment:
This can be written more cleanly in scss, something like:

```css
ul, ol {
  margin-top: 0.25rem;
  padding-left: 1.5em;
  
  li {
    ...
  }
}
```